tools/events/ion_event_equivalence.cpp

```diff
@@ -340,11 +340,11 @@ BOOL ion_compare_sets_embedded(ION_EVENT_EQUIVALENCE_PARAMS, size_t *expected_le
                     ION_ASSERT(ION_COMPARISON_TYPE_ARG == COMPARISON_TYPE_NONEQUIVS,
                                "Invalid embedded documents comparison type.");
                     if (step_expected == 1 && step_actual == 1) {
-                        ION_EXPECT_FALSE(step_expected == 1 && step_actual == 1,
+                        ION_EXPECT_FALSE_FOR_NON_EQUIVS(step_expected == 1 && step_actual == 1,
                                          "Both embedded streams are empty stream in a non-equivs set.")
```

Comment:
The indentation of this line should align with the first character after the open-parenthesis on the previous line, as it did before.

Reply:
Sure, I made the line 343 a new line since line 344 exceeds the right side boundary.